spec/lib/data_update_scripts/create_profile_fields_spec.rb

```diff
@@ -0,0 +1,35 @@
+require "rails_helper"
+require Rails.root.join("lib/data_update_scripts/20200901040521_create_profile_fields.rb")
+
+def profile_field_and_group_count
```

Comment:
Just a little helper so I can verify both in a single statement, which is useful for the `change` matcher.